www/js/con-author.js

```diff
@@ -795,6 +890,22 @@ define([
 					else
 						this.applyDirectives(this.authorPM.process(this.currentID, 'equation', this._model.authored.getEquation(this.currentID), true));
 				}
+				//sessionStorage contains the schema table, load options from schema table
+				var schemasList = [];
+				var schemaTabOb = JSON.parse(sessionStorage.getItem("schema_tab_topo"));
+				for(var parSchema in schemaTabOb){
+					schemaTabOb[parSchema].forEach(function(atomSchema){
+								var sname = atomSchema["Name"];
+								var obj = {value: sname, label: sname};
+								schemaWidget.addOption(obj);
```

Comment:
There's a bug with this: we add the options every time this function is called, but we never delete them.  So an extra copy of the schemas is being added every time the node editor opens.  

Perhaps adding the options should be done once when the tutor first loads (in index.php or main.js?) instead of when the node editor opens, because we don't expect the list of schemas to change during a session.